apis/apps/v1alpha1/apicast_types.go

```diff
@@ -92,6 +93,16 @@ type APIcastSpec struct {
 	// ServiceConfigurationVersionOverride contains service configuration version map to prevent it from auto-updating.
 	// +optional
 	ServiceConfigurationVersionOverride map[string]string `json:"serviceConfigurationVersionOverride,omitempty"` // APICAST_SERVICE_${ID}_CONFIGURATION_VERSION
+	// HttpsPort controls on which port APIcast should start listening for HTTPS connections. If this clashes with HTTP port it will be used only for HTTPS.
+	// +optional
+	HTTPSPort *int32 `json:"httpsPort,omitempty"` // APICAST_HTTPS_PORT
+	// HTTPSVerifyDepth defines the maximum length of the client certificate chain.
+	// +kubebuilder:validation:Minimum=0
+	// +optional
+	HTTPSVerifyDepth *int `json:"httpsVerifyDepth,omitempty"` // APICAST_HTTPS_VERIFY_DEPTH
```

Comment:
This should be `*int32` (or *int64) because int is ambiguos depending on the platform, as per API Conventions (see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md, "primitive types" subsection)
I'd go for *int64, and also for HTTPSPort, to be consistent with the other defined fields in the CR to not have a mix of sizes (even if not that much length is required).

Reply:
Ok, changing to `*int64`. 

HTTPSPort is better to be `*int32` as it is the type required by `corev1.ContainerPort` type's `ContainerPort` field

Reply:
:+1: